docs/tutorials/3DTiles-point-cloud-pnts.md

```diff
@@ -167,23 +167,23 @@ to threejs documentation for more information on `Object3D`, `Group`, `Points` a
 Then, we just need to pass this callback to the `{@link C3DTilesLayer}` constructor:
 
 ```js
-    onTileContentLoaded: updatePointCloudSize
```

Comment:
This is deprecated, we should use `C3DTILES_LAYER_EVENTS.ON_TILE_CONTENT_LOADED` instead now. Could you take this opportunity to update it in the tutorial please?

Reply:
Updated, though I'm not exactly sure what can and can't be linked so I just assumed "missing from left side menu of itowns-project.org" => can't be linked